include/swift/AST/ActorIsolation.h

```diff
@@ -48,6 +48,9 @@ class ActorIsolation {
     /// meaning that it can be used from any actor but is also unable to
     /// refer to the isolated state of any given actor.
     Independent,
+    /// Semantically the same as Independent, but drops the static safety check
+    /// that prevents the use of actor-independent mutable state.
+    IndependentUnsafe,
```

Comment:
If `Independent` and `IndependentUnsafe` have the same semantics, then a well-defined program will have identical behavior in both cases. If that's the case, then this should be called `IndependentUnchecked`.

However, if this attribute is meant to allow state to be accessed across actors, leaving it up to the programmer to protect that state via synchronization or same-thread requirements, then `IndependentUnsafe` is the correct term, but it definitely does not have the same semantics.

Reply:
The latter is intended. I'll reword the comment.